packages/design-picker/src/available-designs-config.json

```diff
@@ -163,6 +163,76 @@
 			"preview": "static",
 			"features": []
 		},
+		{
+			"title": "Marl",
+			"slug": "marl",
+			"template": "blockbase",
+			"theme": "marl",
+			"categories": [
+				{ "slug": "featured", "name": "Featured" },
+				{ "slug": "store", "name": "Store" }
```

Comment:
Adding the `store` category doesn't seem to do anything so I'm not sure if we need it. I'm still not sure where the categories themselves are coming from...I have yet to find "Link in Bio" listed anywhere, for example.

Reply:
https://github.com/Automattic/wp-calypso/issues/60152 has info about this based on some of the research I did a couple weeks back.

Reply:
Aha, so this PR is not required, because the designs come from a particular tag on the Theme Showcase site; is that correct?

Reply:
As far as I understand it from my research, yes. 😄